src/components/Common/Module/ExpressionsHelp.vue

```diff
@@ -258,6 +265,8 @@
 </template>
 
 <script>
+import i18n from '../../../i18n'
+
 export default {
   data () {
```

Comment:
We can use a i18n prefix here to avoid typing `expressions-help` in every translation
We'll go through this in our sync